stock_available/views/product_product_view.xml

```diff
@@ -60,16 +60,11 @@
         <field name="model">product.product</field>
         <field name="inherit_id" ref="stock.view_stock_product_tree" />
         <field name="arch" type="xml">
-            <tree position="attributes">
-                <attribute
-                    name="decoration-danger"
-                >virtual_available&lt;0 or immediately_usable_qty&lt;0</attribute>
-                <attribute
-                    name="decoration-info"
-                >virtual_available&gt;=0 or immediately_usable_qty&gt;0</attribute>
-            </tree>
             <field name="virtual_available" position="after">
-                <field name="immediately_usable_qty" />
+                <field
```

Comment:
@ChrisOForgeFlow @LoisRForgeFlow @pedrobaeza I think this was missing the group, no ?

Reply:
Uhm, yes, according latest upstream code since https://github.com/odoo/odoo/pull/136603